sphinx_gallery/gen_rst.py

```diff
@@ -741,7 +757,7 @@ def _get_last_repr(capture_repr, ___):
             last_repr = None
             repr_meth = None
         else:
-            if isinstance(last_repr, str):
+            if isinstance(last_repr, (str, bytes)):
```

Comment:
```suggestion
            if isinstance(last_repr, str) or ininstance(last_repr, bytes) and meth in bytes_repr:
```
Can be an option (and then define `bytes_repr` somewhere, currently only including `_repr_png_`.